### PR TITLE
Handle windows path separator

### PR DESCRIPTION
**Pull request by mattn**
handle backslash on windows.

### CODE REVIEW

File: matcher.c [hunk 1]
```diff
@@ -21,6 +21,12 @@ typedef struct
   int     never_show_dot_files;   // boolean
 } matchinfo_t;
 
+#ifdef _WIN32
+#define IS_PATHSEP(c) (c == '\\' || c == '/')
+#else
+#define IS_PATHSEP(c) (c == '\\')
```
Comment (mleech) [inline matcher.c]: shouldn't this be 

```
#define IS_PATHSEP(c) (c == '/')
```

for the non-windows case?

Comment (mattn) [inline matcher.c]: Oops, sorry. I'll fix soon.

Comment (mattn) [inline matcher.c]: Done, pushed forcely.

**Comment by mattn** [comment]
Sorry for noise. Done to fix.

**Pull request merged**